gtk/src/default/unity-to-metacity.sh

```diff
@@ -0,0 +1,135 @@
+#!/usr/bin/env bash
+
+for FLAVOUR in default dark light; do
```

Comment:
I am not sure if I got this script completely, but I wonder if this is the right place for such script. If it applies to default, light and dark, is there a reason to keep it inside the default folder only?

Reply:
The `unity-to-metacity.sh` script is not intended to be run routinely. So, not something that should be executed every build, but only when the unity window assets are modified and the metacity assets require refreshing.

Reply:
@clobrano Anything more you need from me?

Reply:
I am sorry @flexiondotorg, I just missed this message. All ok for me, thank you!